pkg/specgen/generate/container_create.go

```diff
@@ -272,6 +273,19 @@ func createContainerOptions(ctx context.Context, rt *libpod.Runtime, s *specgen.
 	// Security options
 	if len(s.SelinuxOpts) > 0 {
 		options = append(options, libpod.WithSecLabels(s.SelinuxOpts))
+	} else {
+		if pod != nil {
+			// duplicate the security options from the pod
+			processLabel, err := pod.ProcessLabel()
+			if err != nil {
+				return nil, err
```

Comment:
This should probably be nonfatal, making pods without an infra container is perfectly valid.

Reply:
There will be no SELinux label though, although we have to figure out how to get one.

Reply:
I checked inside of the code where it checks for an infra container, if no infra then it will return "" for process label, which will get SELinux to fall back to default, which is to label the containers differently.
Added a test for this use case.